schema_utils.py

```diff
@@ -91,7 +88,7 @@ def normalize_against_schema(
     Raises:
         AssertionError. The object fails to validate against the schema.
     """
-    normalized_obj = None # type: Any
+    normalized_obj = None
```

Comment:
I think using Any here is a good option. Reason is that incoming object here can have any type.

Reply:
My bad, didn't notice the types have been removed. Resolving.